MAINTAINERS.md

```diff
@@ -11,3 +13,26 @@
 |[sunyuan3](https://github.com/sunyuan3)|Yuan Sun|yile.sy@alibaba-inc.com |Alibaba Group|
 |[HusterWan](https://github.com/HusterWan)|Michael Wan|zirenwan@gmail.com|Alibaba Group|
 |[ZYecho](https://github.com/ZYecho)|Yue Zhang|zy675793960@yeah.net|ISCAS|
+|[fuweid](https://github.com/fuweid)|Wei Fu|yuge.fw@alibaba-inc.com|Alibaba Group|
```

Comment:
is it good to show the group email here? :see_no_evil:

Reply:
I think it is OK to show this.

Reply:
ok